imap/caldav_alarm.c

```diff
@@ -1132,6 +1132,145 @@ static int process_valarms(struct mailbox *mailbox,
 }
 
 #ifdef WITH_JMAP
+static int move_to_mailboxid(struct mailbox *srcmbox,
```

Comment:
nit: is move_to_mailbox introduced in this commit, but used in the previous commit already? If so, they should be one commit.

Reply:
This was committed before it was used in process_futurerelease()